__tests__/gatsby-theme-project-portal/SearchBar.spec.tsx

```diff
@@ -58,15 +83,25 @@ describe("SearchBar", () => {
 
   it("sets the htmlFor attribute on the label element", () => {
     const { getByText } = render(
-      <SearchBar label="Test Label" onChange={onChange} />
+      <SearchBar
+        label="Test Label"
+        onChange={onChange}
+        id={"search"}
+        placeholder={"Type to filter posts..."}
+      />
     )
     //check rendered component has attribute id = search-label
     expect(getByText("Test Label")).toHaveAttribute("id", "search-label")
   })
 
   it("applies the style prop correctly to the input element", () => {
     const { getByLabelText } = render(
-      <SearchBar label="Test Label" onChange={onChange} />
+      <SearchBar
+        label="Test Label"
+        onChange={onChange}
+        id={"search"}
+        placeholder={"Type to filter posts..."}
+      />
     )
     expect(getByLabelText("Search")).toHaveStyle({ height: "62%" })
```

Comment:
Can you say a bit more about this test? I may be misunderstanding it.
It looks like you're checking the styling – I think that would better be handled using a snapshot test, as the existing test is likely to be brittle if we change the styling. If so, I'd remove it here and reintroduce this kind of testing in the PR where we do the modal.

Reply:
Heya, so I updated this test because I changed the props required for the searchbar component. The test will produce an error since all the required props aren't there. The id really needs to be a required prop since that determines focus.

As for the test itself, that was created in a different pr; do we want to look over our tests in another PR?